internal/compiler/transforms/index.ts

```diff
@@ -36,6 +36,7 @@ import scopedRomeTransform from "./compileForBundle/scopedRomeTransform";
 import asyncImportTransform from "./compileForBundle/asyncImportTransform";
 import inlineEnv from "./compileForBundle/inlineEnv";
 import {commentInjectorVisitor, variableInjectorVisitor} from "./helpers";
+import {default as cssPrefix} from "@internal/compiler/transforms/compile/css-handler/prefix";
```

Comment:
We should differentiate this from the other imports that are just plain objects to be a bit clrearer

```suggestion
import {default as createCSSPrefixVisitors} from "./compile/css-handler/prefix";
```